ios/Makefile

```diff
@@ -1,9 +1,18 @@
-archive_path = PolyPod.xcarchive
+archive_path = ./build/PolyPod.xcarchive
```

Comment:
Any suggestions to improve Makefile are welcome, I am a newbie with Makefile format :)

Reply:
What do you want to do?

Reply:
I had two things:
- Using the environment variables from GitHub secrets, I didn't find other solution than having those bound to a local Makefile variable and then use `$(value variable)`. I guess this could be the acceptable way of doing things.
- I would've liked to have `set -o pipefile` in Makefile, but I couldn't make it work. Not sure if it is even possible,

Reply:
I think you're good with that.